src/LowercaseSentenceTokenizerTest.java

```diff
@@ -16,9 +16,16 @@ void testTokenizeWithNoCapitalizationOrPeriod() {
     }
 
     // Wave 2
-    /*
-     * Write your test here!
-     */
+    @Test
+    void testTokenizeWithNoSpaces() {
+        LowercaseSentenceTokenizer tokenizer = new LowercaseSentenceTokenizer();
+        Scanner scanner = new Scanner("This is a  sentance      without extra spaces");
+        List<String> tokens = tokenizer.tokenize(scanner);
+
+        assertEquals(List.of("This", "is", "a", "sentance", "without", "extra", "spaces"), tokens);
+    }
```

Comment:
It looks like this test does not work with your code. You can see on the GitHub page that it did not pass, and if you run it locally you should see a problem as well. I think the logic in your tokenizer is correct - the issue is with the test itself. Can you see the problem with your expected list in the assertion (hint: capitalization).